arrow-libs/optics/arrow-optics/src/main/kotlin/arrow/optics/std/tuple.kt

```diff
@@ -103,6 +207,14 @@ fun <A, B> Tuple3.Companion.pTraversal(): PTraversal<Tuple3<A, A, A>, Tuple3<B,
 /**
  * [Traversal] to focus into the first, second and third value of a [arrow.core.Tuple3]
  */
+@Deprecated(
+  "Use the tripleTraversal function exposed in the Traversal' companion object",
+  ReplaceWith(
+    "Traversal.tripleTraversal()",
+    "arrow.optics.Traversal"
+  ),
+  DeprecationLevel.WARNING
+)
 fun <A> Tuple3.Companion.traversal(): Traversal<Tuple3<A, A, A>, A> = pTraversal()
 
 /**
```

Comment:
If `pTraversal` for `Tuple4` -> `Tuple5` is added to `Traversal` than all the other ones here have to be deprecated also.

Reply:
thanks @nomisRev for the review! applied the comments 👍